patternfly/patternfly-elements
# Add automatic Sass globbing to pfe-sass

## Add automatic Sass globbing to pfe-sass

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Add build tasks for pfe-sass
* Move glob files to being dynamically built
* Add maps

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

-  `npm run demo`
- Ensure no visual regressions, no errors in console

#### Requirements
Your repository infrastructure updates should work for at least:

### Ready-for-merge Checklist

- [x] Confirm no regressions in compiled assets
- [x] Was this feature demo'd?


**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
